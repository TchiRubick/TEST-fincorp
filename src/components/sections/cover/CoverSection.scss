@import '../../style.scss';

.section-cover {
  display: flex;
  flex-direction: column;
  min-height: 60vh;
  justify-content: space-between;

  .section-cover__greeting {
    display: flex;
    flex-direction: column;

    .section-cover__greeting-subtitle {
      font-weight: bold;
      color: $color-subtext;
      text-transform: uppercase;
      margin-bottom: 0.5rem;
    }

    .section-cover__greeting-title {
      @include title;
    }
  }

  .section-cover__title {
    text-align: center;
    display: flex;
    flex-direction: column;

    .section-cover__title__main {
      @include title;
      margin-bottom: 1rem;
    }

    .section-cover__title__sub {
      color: $color-subtext;
      font-weight: bold;
      font-size: 11px;
    }
  }
}