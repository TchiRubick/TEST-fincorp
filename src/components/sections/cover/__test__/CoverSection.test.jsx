import React from 'react';
import ReactDom from 'react-dom';
import { cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';

import CoverSection from '../CoverSection';

afterEach(cleanup);

it('renders good', () => {
  const div = document.createElement('div');

  ReactDom.render(<CoverSection />, div);
});

it('as expected snapshot 1', () => {
  const tree = renderer.create(<CoverSection />).toJSON();
  expect(tree).toMatchSnapshot();
});
