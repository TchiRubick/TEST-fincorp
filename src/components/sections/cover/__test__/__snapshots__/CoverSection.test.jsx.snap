// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`as expected snapshot 1 1`] = `
<div
  className="comp-container"
>
  <div
    className="section-cover"
  >
    <div
      className="section-cover__header"
    >
      <div
        className="comp-header"
      >
        <img
          alt="logo"
          src="logo.png"
        />
        <span>
          For You & Family
        </span>
        <span>
          For Business
        </span>
      </div>
    </div>
    <div
      className="section-cover__greeting"
    >
      <span
        className="section-cover__greeting-subtitle"
      >
        hello, we are fincorp
      </span>
      <span
        className="section-cover__greeting-title"
      >
        Insurance 
        <br />
        made easy.
      </span>
      <div>
        <button
          className="comp-button-main"
          type="button"
        >
          Get your free Quote
        </button>
      </div>
    </div>
    <div
      className="section-cover__title"
    >
      <span
        className="section-cover__title__main"
      >
        A new take on insurance
      </span>
      <span
        className="section-cover__title__sub"
      >
        Great for individuals and business
      </span>
    </div>
  </div>
</div>
`;
